cli.js

```diff
@@ -17,6 +17,7 @@ var cli = meow({
 		'',
 		'Options',
 		'  --json     Output the result as JSON',
+		'  --bytes    Output the result in megabytes per second',
 		'  --verbose  Output more detailed information'
 	]
 });
```

Comment:
Yeah, sure.

It doesn't really matter in this case as we don't accept input. In those cases if you do `$ cmd --flag input`, is the `input` the value to `--flag` or the `cmd` input. That's also why I usually prefer `--flag=value`, because there's no ambiguity.